docs/user-guide/admin-cli.md

```diff
@@ -58,12 +386,39 @@ vol        datastore1  detached
 Note that the that the choices are given in a comma separated list with no spaces, and are shown in
 the help given above with `vmdkops_admin ls -h`.
 
-# policy
+### Set
+Modify attribute settings on a given volume. The volume is identified by its name and datastore,
+for example if the volume name is `container-vol` then the volume is specified as "container-vol@datastore-name".
+The attributes to set/modify are specified as a comma separated list as "<attr1>=<value>, <attr2>=<value>....". For example,
+a command line would look like this.
+
+```bash
+$ vmdkops-admin set --volume=<volume@datastore> --options="<attr1>=<value>, <attr2>=<value>, ..."
+```
+
+The volume attributes are set and take effect only the next time the volume attached to a VM. The changes do not impact any VM
+thats currently using the volume. For the present, only the "access" attribute is supported to be modified via this command, and
```

Comment:
Could you file a issue to track this? If you don't want to fix this in current PR?